create-liff-app.ts

```diff
@@ -96,14 +83,25 @@ export async function createLiffApp(answers: Answers) {
     fs.writeFileSync(path.join(root, 'package.json'), JSON.stringify(pkg, null, 2));
 
     // create .env file
-    const content = `${envPrefix[templateName]}LIFF_ID=${liffId}`;
-    const envFileName = envFileNameVariant[templateName] || '.env';
+    const content = `${templateConfig.envPrefix}LIFF_ID=${liffId}`;
+    const envFileName = templateConfig?.envFileNameVariant || '.env';
     fs.writeFileSync(path.join(root, envFileName), content);
 
     // install
     const isYarn = packageManager === 'yarn';
-    if (installNow) {
-      await install(root, isYarn);
+    const { dependencies, devDependencies, tsDevDependencies } = templateConfig;
+    if (isTypescript) devDependencies.push(...tsDevDependencies);
+
+    console.log(`\n${installNow ? 'Installing' : 'Updating'} dependencies:`);
```

Comment:
> remove the install now option and install the dependencies by default list create-next-app does. This is lesserror-prone.

As you mentioned, we can delete `installNow` I think.

Reply:
ok, I'll remove this feature.